### PR TITLE
feat(migrate): cover the remaining v9 breaking changes

### PR DESCRIPTION
**Pull request by ShaneK**
Issue number: internal

---------

## What is the current behavior?

Currently, `@ionic/migrate` covers 15 of the breaking changes in the v9 upgrade guide. The Angular `moduleResolution` change and `@ionic/core`'s new `exports` field aren't in there, and neither are about a dozen others.

There's also a gap for vanilla apps: `detectFrameworks` only knows `@ionic/angular`, `@ionic/react`, and `@ionic/vue`, so a vanilla app prints "No @ionic/{angular,react,vue} dependency found" and exits. Every migration marked `framework: 'core'` is unreachable for the apps it was written for.

## What is the new behavior?

This PR adds 14 migrations, taking the registry to 29 (12 auto-fix, 17 report-only). The full table is in the new `packages/migrate/docs/v9.md`. The README is version-agnostic now, so the next major gets its own page instead of a rewrite.

We also detect `@ionic/core` as a framework, so vanilla apps run the `core` migrations and get their pin bumped. We gate the version check on the binding package when there's one, because an app on a v9 binding with a stale `@ionic/core` pin is already migrated, and taking the lowest major would re-run the single-shot import rewrite and corrupt it.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

Note: this PR is based off of the end result of [ionic-docs#4620](https://github.com/ionic-team/ionic-docs/pull/4620). Ten of the docs anchors these migrations link to only exist on that branch.

Two v9 changes stay uncovered on purpose, with the reasoning in `docs/v9.md`: React Router's `/*` suffix for nested routes, and which components Angular 22's `OnPush` default affects. We do flag the Angular 22 version itself.

### CODE REVIEW

**Review by codeCraft-Ritik — COMMENTED** [review]
Really thorough PR, @ShaneK. A few things I appreciated while going through this:

The vanilla app gap fix — detectFrameworks() excluding core meant every framework: 'core' migration was dead code for the apps it was written for. Adding @ionic/core to the detection map and introducing sourceMajor() is the right solution. The binding-over-core priority prevents the corruption scenario where a v9 Angular app with a stale @ionic/core@^8 pin would re-select destructive migrations.

core-modal-handle's brace-balancing — Scoping handleBehavior checks to the enclosing options object (via enclosingObject()) is a thoughtful touch. Without it, one opted-out sheet modal would silence every other sheet in the file. The Vue .vue test case (template opt-out shouldn't suppress script block findings) is a great edge case to cover.

core-form-structure's rule ordering — Putting the slot rules before the broader wrapper rule so the guide's own example selector (ion-input .input-wrapper .native-wrapper [slot="start"]) gets the .input-start answer instead of .input-control shows attention to the developer experience of the report output.

ScanMatch union type — Letting a single migration emit per-finding docsUrl overrides is a clean abstraction. core-select-events linking ionChange findings to one anchor and selected role findings to another is exactly the kind of granularity that saves someone's debugging time.

Minor note: the core-textarea-height scan for 56px scoped to files mentioning ion-textarea is a pragmatic tradeoff — 56px is too common to flag globally, but it does mean a stylesheet that sets the height on a selector like .my-textarea-wrapper without the tag name would be missed. Probably fine for real-world usage though.

Solid work taking the registry to 29 with full fixture-backed tests. 👏

File: packages/migrate/docs/v9.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Migrating v8 to v9
+
+What `npx @ionic/migrate` covers for the Ionic 8 to Ionic 9 upgrade, and what it
+leaves to you. The full list of breaking changes, with before/after examples, is
```
Comment (thetaPC) [inline packages/migrate/docs/v9.md]: The first sentence sounds a bit weird to me. Maybe: "This page covers what `npx @ionic/migrate` handles for the Ionic 8 to Ionic 9 upgrade, and what it leaves to you."

File: packages/migrate/docs/v9.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+in the [Ionic 9 upgrade guide](https://ionicframework.com/docs/updating/9-0).
+Every finding the tool prints links to the section of that guide it came from.
+
+## Coverage
+
+| Change | Framework | Mode |
```
Comment (thetaPC) [inline packages/migrate/docs/v9.md]: Might be worth creating separate tables for each framework and each table would have it's own title. It would be beneficial to devs who just want to review specific frameworks that are relevant to them.

```
### Angular

"table for angular"

### React

....
```

File: packages/migrate/src/migrations/v9/angular-ionic-module.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import type { Finding, Migration } from '../../types.js';
+
+/**
+ * `IonicModule` is deprecated in Ionic 9 in favor of `provideIonicAngular()`,
+ * which works in both standalone and NgModule apps. Report-only: it still
+ * works, and the rewrite is structural (the entry moves from a module's
+ * `imports` to its `providers`, with any `forRoot()` config).
+ *
+ * See https://ionicframework.com/docs/updating/9-0#ionicmodule-deprecation
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/angular-ionic-module.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#ionicmodule-deprecation
```

File: packages/migrate/src/migrations/v9/angular-module-resolution.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+ * `@ionic/angular/lazy`.
+ *
+ * The `angular-standalone-imports` migration introduces those subpath imports,
+ * so without this fix it can leave an app that no longer compiles.
+ *
+ * Edited as text rather than through `JSON.parse`: Angular's generated
+ * `tsconfig.json` carries `//` comments, which are valid JSONC but not JSON, and
+ * round-tripping through `JSON.parse`/`stringify` would silently drop them.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#module-resolution
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/angular-module-resolution.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#module-resolution
```

File: packages/migrate/src/migrations/v9/angular-typescript.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import { createDepsMigration } from '../../ast/deps-migration.js';
+
+/**
+ * Ionic 9 requires TypeScript 5.4 or later, matching Angular 18's own minimum.
+ * A higher pin is left alone (Angular 21 wants 5.9, Angular 22 wants 6.0).
+ *
+ * Pinned with a tilde: Angular's peer ranges are narrow (18 accepts `>=5.4 <5.6`),
+ * and `^5.4.0` resolves to 5.9.x, which nothing below Angular 20 accepts.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#typescript
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/angular-typescript.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#typescript
```

File: packages/migrate/src/migrations/v9/angular-version.ts [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+import type { Finding, Migration } from '../../types.js';
+import { V9_DOCS } from './docs.js';
+
+/**
+ * Report-only checks on the Angular toolchain around an Ionic 9 upgrade. Neither
+ * is ours to rewrite: `ng update` owns the Angular version, and
+ * `@ionic/angular-toolkit` versions on its own release line, so there is no
+ * major this tool could safely write for it.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#angular
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/angular-version.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#angular
```

File: packages/migrate/src/migrations/v9/angular-version.ts [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+            `Angular ${major} raises the minimum Node.js version. Check your CI and local runtime`,
+            `${V9_DOCS}#nodejs`
+          )
+        );
+      }
+    }
+
+    if (findDependency(parsed.pkg, TOOLKIT)) {
+      findings.push(
+        at(`update ${TOOLKIT} to its latest release. It versions separately from @ionic/angular`, `${V9_DOCS}#angular`)
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/angular-version.ts:71]: Should it be capitalized?

```suggestion
        at(`Update ${TOOLKIT} to its latest release. It versions separately from @ionic/angular`, `${V9_DOCS}#angular`)
```

File: packages/migrate/src/migrations/v9/core-browserslist.ts [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Ionic 9 raises its supported browser floors. The guide prints a replacement
+ * `.browserslistrc` block, but this raises the version numbers in place:
+ * overwriting the file would drop comments and queries unrelated to Ionic.
+ *
+ * Only entries this owns are touched - a named browser with a `>=` floor below
+ * Ionic 9's.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#browser-support
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-browserslist.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#browser-support
```

File: packages/migrate/src/migrations/v9/core-capacitor.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
+import { isPlainSemverRange, parseMajor } from '../../detect.js';
+import { findDependency, readPackageJson } from '../../ast/package-json.js';
+import type { Migration } from '../../types.js';
+
+/**
+ * Ionic 9's `isCapacitorNative` relies solely on `Capacitor.isNativePlatform()`,
+ * added in Capacitor 3. Report-only: the fix is a Capacitor upgrade.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#capacitor
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-capacitor.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#capacitor
```

File: packages/migrate/src/migrations/v9/core-deps.ts [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+import { IONIC_V9_VERSION } from '../../versions.js';
+
+/**
+ * Raise a directly declared `@ionic/core` to v9, for a vanilla app and for a
+ * framework app that pins `@ionic/core` itself and would otherwise resolve the
+ * v8 build next to a v9 binding. Nothing happens when `@ionic/core` is only a
+ * transitive dependency, since `createDepsMigration` skips packages the
+ * manifest does not declare.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#core
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-deps.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#core
```

File: packages/migrate/src/migrations/v9/core-floating-label.ts [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+ * floats just because the control has slotted content. It floats on focus or a
+ * value.
+ *
+ * Only reported when the element has slotted content: the change is
+ * invisible without it, and floating labels are common enough that flagging
+ * every one would be noise.
+ *
+ * Report-only - whether the old look mattered is a design call.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#input-floating-label-behavior
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-floating-label.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#input-floating-label-behavior
```

File: packages/migrate/src/migrations/v9/core-form-structure.ts [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+
+/**
+ * `ion-input`, `ion-textarea`, and `ion-select` reorganized their internal DOM
+ * to support floating labels alongside slotted content. Stylesheets reaching
+ * into that structure stop matching, with no error.
+ *
+ * Report-only: which replacement is right depends on what the rule was for, and
+ * `part="inner"` on `ion-select` has no replacement at all.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#input-internal-dom-structure-changes
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-form-structure.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#input-internal-dom-structure-changes
```

File: packages/migrate/src/migrations/v9/core-modal-handle.ts [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+ * `"none"`. A sheet modal's handle becomes focusable, and activating it cycles
+ * the sheet through its breakpoints.
+ *
+ * Scoped to sheet modals: without `breakpoints`/`initialBreakpoint` there is no
+ * handle for the new default to affect. A modal that already sets
+ * `handleBehavior` is left alone.
+ *
+ * Report-only: whether the old inert handle was intentional is the app's call.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#modal
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-modal-handle.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#modal
```

File: packages/migrate/src/migrations/v9/core-package-exports.ts [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+ * TypeScript's `bundler`/`node16`/`nodenext` resolution.
+ *
+ * Scanned as text rather than through ts-morph because the apps most likely to
+ * reach into `@ionic/core` directly are vanilla ones, whose sources are `.js`
+ * and `.html` - neither of which ts-morph loads (`allowJs: false`).
+ *
+ * Report-only: which supported subpath replaces an internal path depends on
+ * what was being imported.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#package-exports
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-package-exports.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#package-exports
```

File: packages/migrate/src/migrations/v9/core-select-events.ts [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+import { V9_DOCS } from './docs.js';
+
+/**
+ * Two `ion-select` changes that fail silently:
+ *
+ *   - `ionChange` now fires only on an actual change. The `alert` and
+ *     `action-sheet` interfaces used to emit on every confirmation, so a handler
+ *     doing work on re-confirmation of the same value stops running.
+ *   - The `action-sheet` interface no longer assigns the `selected` role, so
+ *     code reading that role off `ionActionSheetDidDismiss` now sees
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-select-events.ts]:
```suggestion
 *     code reading that role off `ionActionSheetDidDismiss` now reads
```

File: packages/migrate/src/migrations/v9/core-select-events.ts [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+ *     `undefined`.
+ *
+ * Scoped to `ion-select` itself, so `ionChange` on the many other components
+ * that emit it is not reported. Report-only: only the handler's body says
+ * whether it depended on the old firing pattern.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#select
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-select-events.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#select
```

File: packages/migrate/src/migrations/v9/core-swipe-back-config.ts [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+
+/**
+ * In React and Vue the `swipeBackEnabled` config is read once, when the outlet
+ * mounts. Toggling it at runtime no longer does anything; use the
+ * `swipeGesture` property on `ion-router-outlet` instead.
+ *
+ * Reported for every usage: telling the dynamic ones apart means proving the
+ * value never changes after mount, which a source scan cannot do.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#swipebackenabled-config-behavior-change
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-swipe-back-config.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#swipebackenabled-config-behavior-change
```

File: packages/migrate/src/migrations/v9/core-swipe-back-config.ts [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+  framework: 'core',
+  fromMajor: 8,
+  toMajor: 9,
+  status: 'stable',
+  docsUrl: 'https://ionicframework.com/docs/updating/9-0#swipebackenabled-config-behavior-change',
+
+  detect(ctx) {
+    return scanLines(ctx, SOURCE_GLOBS, (line) =>
+      SWIPE_BACK.test(line)
+        ? 'in React and Vue, swipeBackEnabled is read once at outlet mount now. If you toggle it at runtime, use the swipeGesture property'
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-swipe-back-config.ts:27]:
```suggestion
        ? 'In React and Vue, swipeBackEnabled is read once at outlet mount now. If you toggle it at runtime, use the swipeGesture property.'
```

File: packages/migrate/src/migrations/v9/core-textarea-height.ts [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * The Material Design minimum height of `ion-textarea` went from `56px` to
+ * `72px`, and is now the same regardless of `fill` or `labelPlacement`. Layout
+ * built around the old number silently shifts.
+ *
+ * Report-only, and scoped to stylesheets that mention `ion-textarea`: `56px` is
+ * common enough that flagging it everywhere would be noise. A stylesheet that
+ * never names the element is missed.
+ *
+ * See https://ionicframework.com/docs/updating/9-0#minimum-height-change
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-textarea-height.ts]:
```suggestion
 * Refer to https://ionicframework.com/docs/updating/9-0#minimum-height-change
```

File: packages/migrate/src/migrations/v9/core-textarea-height.ts [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+    const findings: Finding[] = [];
+    for (const filePath of ctx.glob(STYLE_GLOBS)) {
+      const text = ctx.readFile(filePath);
+      if (text === undefined || !TEXTAREA.test(text)) continue;
+      text.split('\n').forEach((line, i) => {
+        if (OLD_HEIGHT.test(line)) {
+          findings.push({
+            filePath,
+            line: i + 1,
+            detail: 'ion-textarea is 72px tall in md mode now, not 56px. Update or override this value',
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/core-textarea-height.ts:36]:
```suggestion
            detail: 'ion-textarea is 72px tall in md mode now, not 56px. Update or override this value.',
```

File: packages/migrate/src/migrations/v9/react-router-6-code.ts [hunk 21]
```diff
@@ -47,6 +47,10 @@ const COMPONENT_REMOVED: ReportedChange = {
   detail: `Route "component" prop removed. Use "element" with JSX`,
   docsUrl: `${V9_DOCS}#route-definition-changes`,
 };
+const CHILDREN_REMOVED: ReportedChange = {
+  detail: `Route content passed as children no longer renders. Move it into the "element" prop`,
```
Comment (thetaPC) [inline packages/migrate/src/migrations/v9/react-router-6-code.ts:51]:
```suggestion
  detail: `Route content passed as children no longer renders. Move it into the "element" prop.`,
```

File: packages/migrate/README.md [hunk 22]
```diff
@@ -41,44 +41,12 @@ Every breaking change is one of two kinds:
...
-  Capacitor 2 app reports web from `isPlatform('capacitor')` and `'hybrid'`.
-  Upgrade to Capacitor 7 or later.
-- In React and Vue the `swipeBackEnabled` config is read once when the outlet
-  mounts. If you toggle it at runtime, move to the `swipeGesture` prop on
-  `ion-router-outlet`. Setting it once at startup still works, so most apps need
-  no change, which is why we don't flag it.
+### Coverage
+
+Each major upgrade has its own page, listing every change the tool covers, which
+kind it is, and what it leaves to you:
```
Comment (thetaPC) [inline packages/migrate/README.md]: Sounds weird to me, maybe:


```suggestion
Each major upgrade has its own page, listing every change the tool covers, how it is classified, and the changes left for you to make manually:
```

**Review by thetaPC — APPROVED** [review]
LGTM, just grammar

File: packages/migrate/src/migrations/v9/angular-ionic-module.ts (hunk 3 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/angular-ionic-module.ts]: Good catch! Done

File: packages/migrate/src/migrations/v9/angular-version.ts (hunk 7 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/angular-version.ts:71]: Nope, lowercase is deliberate here. Report prints these as `path:line - detail`, so they read as annotations rather than sentences, and they only take a capital when the first word is an identifier that has one, like `Route "component" prop removed` or `IonRedirect removed`. The toolkit one starts on a verb, same as `remove ~ prefix from ...`, so it stays lowercase.

File: packages/migrate/src/migrations/v9/core-swipe-back-config.ts (hunk 18 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-swipe-back-config.ts:27]: This one's the same as [the toolkit finding](https://github.com/ionic-team/ionic-framework/pull/31355#discussion_r3786263646): the details print as annotations after the `path:line - ` prefix, and none of the ~30 in the registry ends in a period, so I'd rather keep them uniform.

File: packages/migrate/src/migrations/v9/core-textarea-height.ts (hunk 20 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-textarea-height.ts:36]: I've left the period off here for the same reason as [the toolkit finding](https://github.com/ionic-team/ionic-framework/pull/31355#discussion_r3786263646), the details print as annotations and none of the ~30 in the registry ends in one.

File: packages/migrate/src/migrations/v9/react-router-6-code.ts (hunk 21 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/react-router-6-code.ts:51]: Leaving this one as-is too, same reason as [the toolkit thread](https://github.com/ionic-team/ionic-framework/pull/31355#discussion_r3786263646).

File: packages/migrate/src/migrations/v9/core-select-events.ts (hunk 15 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-select-events.ts]: Yeah, good point. I went with `gets` instead of `reads` and reworded the front half, since "code reading that role off ... now reads" doubled up the verb.

File: packages/migrate/docs/v9.md (hunk 1 above)
Comment (ShaneK) [reply packages/migrate/docs/v9.md]: Yeah, that was a fragment. Done!

File: packages/migrate/docs/v9.md (hunk 2 above)
Comment (ShaneK) [reply packages/migrate/docs/v9.md]: Good call! Split into Angular, React, Vue, and All frameworks, and dropped the Framework column since the heading covers it now. There's a line up top pointing people at their framework's table plus the core one, since the core changes apply on top.

File: packages/migrate/README.md (hunk 22 above)
Comment (ShaneK) [reply packages/migrate/README.md]: Reworded, though I went with "whether it is auto-fixed or report-only" since those are the two names the section right above defines.

File: packages/migrate/src/migrations/v9/angular-module-resolution.ts (hunk 4 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/angular-module-resolution.ts]: Done!

File: packages/migrate/src/migrations/v9/angular-typescript.ts (hunk 5 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/angular-typescript.ts]: Done!

File: packages/migrate/src/migrations/v9/angular-version.ts (hunk 6 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/angular-version.ts]: Done!

File: packages/migrate/src/migrations/v9/core-browserslist.ts (hunk 8 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-browserslist.ts]: Done!

File: packages/migrate/src/migrations/v9/core-capacitor.ts (hunk 9 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-capacitor.ts]: Done!

File: packages/migrate/src/migrations/v9/core-deps.ts (hunk 10 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-deps.ts]: Done!

File: packages/migrate/src/migrations/v9/core-floating-label.ts (hunk 11 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-floating-label.ts]: Done!

File: packages/migrate/src/migrations/v9/core-form-structure.ts (hunk 12 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-form-structure.ts]: Done!

File: packages/migrate/src/migrations/v9/core-modal-handle.ts (hunk 13 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-modal-handle.ts]: Done!

File: packages/migrate/src/migrations/v9/core-package-exports.ts (hunk 14 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-package-exports.ts]: Done!

File: packages/migrate/src/migrations/v9/core-select-events.ts (hunk 16 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-select-events.ts]: Done!

File: packages/migrate/src/migrations/v9/core-swipe-back-config.ts (hunk 17 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-swipe-back-config.ts]: Done!

File: packages/migrate/src/migrations/v9/core-textarea-height.ts (hunk 19 above)
Comment (ShaneK) [reply packages/migrate/src/migrations/v9/core-textarea-height.ts]: Done!
